README.md

```diff
@@ -1,7 +1,124 @@
-# vscode-granite
+<img src="/media/title.png" alt="ollamacontine" width="1000"/>
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+In this project,we are building local AI co-pilot using **Granite Code, Ollama, and Continue**,where we utilize a collection of open-source components to implement a feature-rich developer co-pilot in Visual Studio Code, all while ensuring data privacy and compliance with licensing requirements. We are integrating Ollama with Continue to seamlessly work with Visual Studio Code. This integration will assist developers by accelerating the development process, enhancing performance, and improving code quality efficiently.
```

Comment:
- add hyperlinks to Granite Code, Ollama and Continue websites
- add spaces after commas